docs/source/faq.rst

```diff
@@ -77,3 +77,15 @@ If you want to save and resume studies,  it's handy to use SQLite as the local s
     study.optimize(objective)  # The state of `study` will be persisted to the local SQLite file.
 
 Please see :ref:`rdb` for more details.
+
+
+I want to make the behavior of a sampler deterministic.
```

Comment:
I think 'deterministic' seems overclaiming. When we use multiple workers, then the sequence of sampling has randomness due to parallelism.
So, I think the titles like 'How to set random seeds to samplers?' or 'How can I fix random seeds of samplers' are more accurate than the current one.

Reply:
Thank you for your modification.
Sorry for bothering you again, but I think the revised sentence lacks the conclusion.
Can we directly state as follows:
- 'You cannot reproduce the optimization result.'
- 'Optuna does not support the reproduction of parallel optimization for now.'

Reply:
Thank you for your advice. I appended "We recommend executing optimization of a study sequentially if you would like to reproduce the result" to the tail of the paragraph.